### ISSUE TITLE
Extension doesn't seem to run

### ISSUE THREAD

**Issue by freshlesh3** [stelabouras/TubeLister#3]
I have a large list of videos but the extension doesn't run.

**Comment by sirus20x6** [comment]
yeah, clicking the button doesn't do anything. got it from the chrome extension store. no feedback whatsoever

**Comment by 7flash** [comment]
can we fix this?

**Comment by sirus20x6** [comment]
is this extension real or some sort of a scam?

**Comment by freshlesh3** [comment]
> is this extension real or some sort of a scam?

It worked at one point, not always consistently but now it is completely non responsive.

**Comment by DovieW** [comment]
someone made a fork and fixed the issue. its not on the chrome store so you have to load it as unpacked (or drop in the zip if the contents are in the root of the zip). no build needed
https://github.com/walbi-malbi/TubeLister

**Comment by christancula** [comment]
I've created similar thing (I suppose) - "WatchlatEE" - separate button to create (temp) playlist, close all YT tabs and reopen previous playlist and copy plaintext list of YT-links.
it's working and listed in store #hopethathelps https://chromewebstore.google.com/detail/watchlatee/kbmdkimgpnojpjaacmecmeilepgcjmkp
